templates/tutorial-tr.md

```diff
@@ -0,0 +1,26 @@
+## Merhaba, Logseq'e hoş geldiniz!
+- Logseq, _bilgi_ yönetimi ve işbirliği için _gizlilik öncelikli_, [açık kaynaklı](https://github.com/logseq/logseq) bir platformdur.
+- Bu, Logseq'in nasıl kullanılacağına dair 3 dakikalık bir eğitimdir. Başlayalım!
+- İşte yararlı olabilecek bazı ipuçları.
+#+BEGIN_TIP
+Herhangi bir bloğu düzenlemek için tıklayın.
+Yeni bir blok oluşturmak için `Enter` tuşuna basın.
+Yeni bir satır oluşturmak için `Shift+Enter` tuşuna basın.
+Tüm komutları göstermek için `/` tuşuna basın.
+#+END_TIP
+- 1. [[Nasıl not alınır?]] adında bir sayfa oluşturalım. O sayfaya gitmek için üzerine tıklayabilir veya sağ kenar çubuğunda açmak için `Shift+Tıkla` yapabilirsiniz! Şimdi hem _Bağlantılı Referanslar_ hem de _Bağlantısız Referanslar_ görmelisiniz.
+- 2. [[Nasıl not alınır?]] sayfasındaki bazı bloklara referans verelim. Sağ kenar çubuğunda açmak için herhangi bir blok referansını 'Shift+Tıkla' yapabilirsiniz.  Sağ kenar çubuğunda bazı değişiklikler yapmayı deneyin, referans verilen bloklar da değişecek!
+    - ((5f713e91-8a3c-4b04-a33a-c39482428e2d)) : Bu bir blok referansıdır.
+    - ((5f713ea8-8cba-403d-ac00-9964b1ec7190)) : Bu başka bir blok referansıdır.
+- 3. Etiketleri destekliyor musunuz?
+    - Tabii ki, bu bir #test etiketi.
+- 4. Yapılacak işler ve öncelikler (todo/doing/done) gibi görevleri destekliyor musunuz?
+    - Evet, `/` tuşuna basın ve favori yapılacaklar anahtar kelimenizi veya önceliğinizi (A/B/C) seçin.
+    - NOW [#A] "Nasıl not alınır?" adlı bir sayfa yarat.
+    - LATER [#A] Not almak ve hayatınızı düzenlemek için Logseq'i nasıl kullanacağınıza dair [:a {:href "https://twitter.com/shuomi3" :target "_blank"} "@shuomi3"] kullanıcısının bu harika videosunu izleyin!
+    {{youtube https://www.youtube.com/watch?v=BhHfF0P9A80&ab_channel=ShuOmi}}
+
+    - DONE Bir sayfa yarat
+    - CANCELED [#C] 1000'den fazla blok içeren bir sayfa yazın
+- Bu kadar! Şimdi bazı notları içe aktarmak için daha fazla madde işareti oluşturabilir veya yerel bir dizin açabilirsiniz!
+- You can also download our desktop app at https://github.com/logseq/logseq/releases
```

Comment:
This phrase is not translated. 😆

Reply:
Opss. I will translate. After translate may I squash this commit and force push?